suite-native/module-device-settings/src/screens/BackupAndPassphrase/CheckBackupCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, CardWithIconLayout, HStack, Text, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { useNavigateToCheckBackup } from '@suite-native/module-check-backup';
+
+export const CheckBackupCard = () => {
+    const { navigateToCheckBackup } = useNavigateToCheckBackup();
+
+    return (
+        <CardWithIconLayout
+            icon="trezorBackup"
+            title={<Translation id="moduleDeviceSettings.checkBackup.title" />}
+        >
+            <VStack spacing="sp16">
+                <Text variant="hint" color="textSubdued">
+                    <Translation id="moduleDeviceSettings.checkBackup.subtitle" />
+                </Text>
+                <HStack spacing="sp8">
```

Comment:
Redundant wrapper. Spacing is unnecessary anyway since it's only applied when multiple children are rendered.  Since flex 1 is applied for the button, this element doesn't do anything, please remove 🙏